.env.example

```
@@ -59,8 +59,8 @@ GOOGLE_CLIENT_SECRET=your_google_client_secret
 # -----------------------------------------------------------------------------
 SMTP_HOST=smtp.ethereal.email
 SMTP_PORT=587
-SMTP_USER=paul46@ethereal.email
-SMTP_PASS=yXcwFudUjewqRAkYXr
+SMTP_USER=jared.raynor0@ethereal.email
+SMTP_PASS=7pPc33ZkrwRT2zrGJq
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SMTP_PASS key should go before the SMTP_PORT key

(UnorderedKey)